scripts/integration-test

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+set -e
+
+export KUBECONFIG=
+export CATTLE_DEV_MODE=yes
+export CATTLE_SERVER_URL="https://$(ip route get 8.8.8.8 | awk '{print $7}'):443"
```

Comment:
Nit: If you want to get rid of the awk I think you could use cut which appears to be installed by default with `bci:golang:1.19`

Reply:
This instance of awk is run inside the rancher/rancher image, which already has awk installed. This code was copied from [rancher](https://github.com/rancher/rancher/blob/b8a90235e0fbba17fce53e0a03019a98325d93f9/scripts/run#L17).
 
I will leave the other usage of awk since it can not easily be done with cut due to the arithmetic operations used to calculate code coverage.